pkg/release/version.go

```diff
@@ -0,0 +1,24 @@
+package release
+
+type Version struct {
+	name      string
+	isChannel bool
+}
```

Comment:
We can further extend it by adding `NewChannelVersion` and `NewReleaseVersion` but I don't want to expose more than necessary.